tools/repl/repl.cpp

```diff
@@ -300,6 +345,9 @@ int main(int argc, char **argv) {
         (readResult == ReadResult::INTERRUPT && code.empty())) {
       // EOF or user exit on non-continuation line.
       llvm::outs() << '\n';
+#if HAVE_LIBREADLINE
+      ::write_history(historyFile.c_str());
```

Comment:
Does readline cap the size of the file?  I don't think we want to let it grow without bound.

Reply:
Sure, thanks for noticing this. There is a `stifle_history` function which can be used to limit this, I will build it.

Reply:
How many entries could be the limit? 500?

Reply:
That's the default for bash.  At 100 chars per expression that's 50k file size, which seems unlikely to fill anybody's homedir, so seems reasonable.

Reply:
Thanks